services/edus/api_v1.json

```diff
@@ -2,17 +2,17 @@
   "openapi": "3.0.3",
   "info": {
     "version": "v1",
-    "title": "CWA-Data Event-Driven-User-Survey (EDUS) API",
+    "title": "CWA-Data Log Viewer API",
```

Comment:
@ioangut / @sorin-stefan-iovita-sap this is EDUS... so it's for the RKI survey system... feel free to correct errors, but don't change it drastically.

For the 'els-verify' please create a new `api_v1.json` within the els-verify folder